python/arcticdb/storage_fixtures/s3.py

```diff
@@ -194,8 +214,12 @@ class MotoS3StorageFixtureFactory(BaseS3StorageFixtureFactory):
     _bucket_id = 0
     _live_buckets: List[S3Bucket] = []
 
+    def __init__(self, use_ssl: bool):
+        self.http_protocol = "https" if use_ssl else "http"
+
+
     @staticmethod
-    def run_server(port):
+    def run_server(port, key_file, cert_file):
```

Comment:
I am not sure I agree with adding new default parameter unless it is necessary
`run_server` should have no external dependency and any one in the team wants it can add it anytime